<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>761</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Registration Block</string>
     </attribute>
     <widget class="QLabel" name="label_welcome">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Heading</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome to Tongji HospitalÔºÅ</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>150</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>150</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sex">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sex</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_age">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dep">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>300</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Consulting Room</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_doc">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Doctor</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_age">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>250</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_sex">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>200</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_dep">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>300</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_doc">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>350</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_submit">
      <property name="geometry">
       <rect>
        <x>435</x>
        <y>400</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_welcome_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Heading</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Please make a registration.</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>130</y>
        <width>401</width>
        <height>331</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_nurse">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>200</width>
        <height>390</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>pushButton_nurse</zorder>
     <zorder>frame</zorder>
     <zorder>label_welcome</zorder>
     <zorder>lineEdit_name</zorder>
     <zorder>label_name</zorder>
     <zorder>label_sex</zorder>
     <zorder>label_age</zorder>
     <zorder>label_dep</zorder>
     <zorder>label_doc</zorder>
     <zorder>lineEdit_age</zorder>
     <zorder>comboBox_sex</zorder>
     <zorder>comboBox_dep</zorder>
     <zorder>comboBox_doc</zorder>
     <zorder>pushButton_submit</zorder>
     <zorder>label_welcome_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Calling and Queuing Block</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>541</width>
        <height>461</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>16</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>80</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>105</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Consulting Room</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Doctor A</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Doctor B</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Doctor C</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Queue Number</string>
       </property>
      </column>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>554</x>
        <y>20</y>
        <width>201</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Doctor Querying Block</string>
     </attribute>
     <widget class="QLabel" name="label_dep_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>171</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Consulting Room</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_dep_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_doc_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Doctor</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_doc_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>255</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>513</width>
        <height>441</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>80</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>85</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="background">
        <color>
         <red>211</red>
         <green>251</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="background">
        <color>
         <red>211</red>
         <green>251</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sex</string>
       </property>
       <property name="background">
        <color>
         <red>211</red>
         <green>251</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Age</string>
       </property>
       <property name="background">
        <color>
         <red>211</red>
         <green>251</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Diag Time</string>
       </property>
       <property name="background">
        <color>
         <red>211</red>
         <green>251</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>B-Scan Time</string>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>211</red>
          <green>251</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HGYT2_CNKI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tongji Hospital Calling System</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
